lightning.install

```diff
@@ -8,6 +8,38 @@
 use Drupal\Core\Entity\Entity\EntityViewDisplay;
 use Drupal\field\Entity\FieldConfig;
 use Drupal\lightning_core\ConfigHelper as Config;
+use Drupal\lightning_core\Element;
+
+/**
+ * Alerts users of Libraries API dependencies.
+ *
+ * A future version of Lightning will remove Libraries API, so
+ * users must add that dependency to composer.json manually.
+ *
+ * @todo Remove this function when Libraries API is no longer shipped with Lightning.
+ */
+function lightning_requirements() {
+  $requirements = [];
+  $libraries_dependencies = [];
+  $enabled_modules = \Drupal::service('extension.list.module')->getAllInstalledInfo();
+
+  foreach ($enabled_modules as $module => $info) {
+    foreach (['libraries:libraries', 'libraries'] as $needle) {
```

Comment:
We can remove a level of nesting here by using array_intersect(), like so:
```php
foreach ($enabled_modules as $module => $info) {
  if (array_intersect($info['dependencies'], ['libraries:libraries', 'libraries'])) {
    $libraries_dependencies[$module] = $info['name'];
  }
}
```

Reply:
yes, much better, thanks!